src/environment/tokens.rst

```diff
@@ -5,91 +5,122 @@ Science Platform Tokens
-This access token can also be used outside the Notebook Aspect to access services from a local machine (for example, to access the TAP service from `TOPCAT`_).
+If you want to access services from a local machine (for example, to access the TAP service from `TOPCAT`_), you will need to create an access token for that purpose.
 
-This page will give you an introduction to the access token, where to find it, how to inspect it, and how to use it in an environment outside the Notebook Aspect.
-Note that the access token has a finite lifetime and is refreshed on each login to the Notebook Aspect.
+This page will give you an introduction to the access token, where to create onefor use from a local machine, and how to use it in an environment outside the Notebook Aspect.
 
-The access token
-================
+The current user interface for managing tokens will look very primitive.
+The functionality is present, but the UI design is not yet complete.
+This will be improved in later development.
 
-The access token is mounted in a known location in your home space.
-To view it, you can simply look a the special file called ``~/.access_token``.
-From within a `terminal`_ execute the following command.
+The Notebook Aspect access token
+================================
 
-.. code-block:: bash
+Whenever you log on to the Notebook Aspect, an access token will be generated for you and stored in your home space in the Notebook Aspect.
+This token is time-limited and will be refreshed when you log out and log back in.
+The token is stored in the file ``~/.access_token`` so that it can be accessed by notebooks and used to authenticate to other Science Platform services.
 
-   cat ~/.access_token
-
-.. _`terminal`: https://jupyterlab.readthedocs.io/en/latest/user/terminal.html
-
-The environment includes a notebook, at ``~/notebooks/notebook-demo/token-info.ipynb``, that allows you to inspect a token.
+The environment includes a notebook, at ``~/notebooks/notebook-demo/token-info.ipynb``, that allows you to inspect that token.
 Executing the two cells in the notebook will provide information about token creation time and lifetime among other things.
 You will note from the above notebook that the token is also provided as the value of the environment variable ``ACCESS_TOKEN``, but the file version should be considered primary.
 
 Using a token outside the Science Platform
 ==========================================
 
-Following is a recipe for an example workflow that uses the access token taken from a JupyterLab session to access the Science Platform TAP service from a local computer using TOPCAT.
-You can apply similar steps to other use cases.
+If you want to use an access token from your local system, you can create a new one.
+You can restrict what that token is able to do to only the services that you need to access, give the token a name, and configure its expiration (or set it to not expire).
```

Comment:
Maybe use a bullet list to?
---
At creation time you can:

- restrict access to only the services you need
- give the token a name
- configure the expiration including setting it to not expire